CHANGES.md

```diff
@@ -1,6 +1,12 @@
 wavesurfer.js changelog
 =======================
 
+6.6.0 (12.03.2023)
+------------------
+- [Regions] Fix undefined content element on remove (#2713)
```

Comment:
[Regions] should be  in a separate list (see other changelogs below for example)